crate/hsm/softhsm2/Cargo.toml

```diff
@@ -12,8 +12,8 @@ description = "Utimaco HSM PKCS#11 loader"
 doctest = false
 
 [dependencies]
-cosmian_kms_base_hsm = { path = "../base_hsm", version = "5.4.1" }
-cosmian_kms_interfaces = { path = "../../interfaces", version = "5.4.1" }
+cosmian_kms_base_hsm = { path = "../base_hsm", version = "5.5.1" }
+cosmian_kms_interfaces = { path = "../../interfaces", version = "5.5.1" }
```

Comment:
Do we need to specify the version (when it is a path?)

Reply:
This is a `cargo publish` requirement